### PR TITLE
Add offset parameter to player.start

### PR DESCRIPTION
**Pull request by cifkao**
Adding an `offset` parameter to enable starting the sequence from a given time.

This will allow me to implement a workaround for my [seeking problem in html-midi-player](https://github.com/cifkao/html-midi-player/issues/2). Related: #502.

### CODE REVIEW

**Comment by notwaldorf** [comment]
Oooh I like this a lot. let me try to restart the travis job to see if the tests failed because of a fluke

**Comment by notwaldorf** [comment]
Argh, I can't restart the build (because "an error occurred" 🤦‍♀️). Would you mind doing a rebase (to trigger a new build?). Thanks!

**Comment by cifkao** [comment]
No problem, done!

File: music/src/core/player.ts [hunk 1]
```diff
@@ -194,7 +195,7 @@ export abstract class BasePlayer {
     if (this.desiredQPM) {
       Tone.Transport.bpm.value = this.desiredQPM;
     }
-    this.currentPart.start();
+    this.currentPart.start(undefined, offset);
```
Comment (notwaldorf) [inline music/src/core/player.ts:198]:
```suggestion
    this.currentPart.start(undefined /* immediately */, offset);
```

**Review by notwaldorf — APPROVED** [review]

**Comment by cifkao** [comment]
@notwaldorf Thanks for merging! Any plans for a release? It would be awesome to have these changes available!

**Pull request merged**